### PR TITLE
Improve CI item views 

### PR DESCRIPTION
**Pull request by HelenaEksler**
#125

### CODE REVIEW

**Comment by HelenaEksler** [comment]
selection_018
selection_017

**Comment by amitaibu** [comment]
Instead of blocks please use Panels

**Comment by HelenaEksler** [comment]
selection_020

File: shoov/modules/custom/shoov_ci_build/shoov_ci_build.views_default.inc [hunk 1]
```diff
@@ -23,8 +23,11 @@ function shoov_ci_build_views_default_views() {
   /* Display: Defaults */
   $handler = $view->new_display('default', 'Defaults', 'default');
   $handler->display->display_options['use_more_always'] = FALSE;
-  $handler->display->display_options['access']['type'] = 'perm';
-  $handler->display->display_options['access']['perm'] = 'administer message types';
+  $handler->display->display_options['access']['type'] = 'role';
+  $handler->display->display_options['access']['role'] = array(
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/shoov_ci_build.views_default.inc:27]: no need to deal with access here

**Comment by amitaibu** [comment]
Merged, please deal with comment in a follow up PR.

**Pull request closed without merging**